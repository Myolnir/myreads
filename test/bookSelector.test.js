import React from 'react';
import { shallow, mount, render } from 'enzyme';
import BookSelector from '../src/components/booksSelector';
import { book }  from './book.fixture';

describe("<BookSelector />", () => {
  it("Should render the component", ()  => {
    const moveToShelf = function(){}
    const bookWrapper = mount(
      <BookSelector
        book={book}
        moveToShelf={moveToShelf}
      />
    )
    expect(bookWrapper.html()).toMatchSnapshot();
  });

});
