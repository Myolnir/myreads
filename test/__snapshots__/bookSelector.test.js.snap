exports[`<BookSelector /> Should render the component 1`] = `"<div class=\"book-shelf-changer\"><select><option value=\"none\" disabled=\"\">Move to...</option><option value=\"currentlyReading\">Currently Reading</option><option value=\"wantToRead\">Want to Read</option><option value=\"read\">Read</option><option value=\"none\">None</option></select></div>"`;
